### DISCUSSION TITLE
code ape : une suggestion disparaît innopinément

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2618] [Idées]
Sur l'outil de recherche de code APE https://mon-entreprise.urssaf.fr/assistants/recherche-code-ape

Je tape <kbd>informatiqu</kbd> et la deuxième suggestion est « Conseil en systèmes et logiciels informatiques 62.02A » comme attendu. Mais si je tape <kbd>informatique</kbd>, cette suggestion disparaît alors qu'elle est toujours aussi pertinente.

![Capture d’écran du 2023-02-28 19-01-10](https://user-images.githubusercontent.com/1730702/221940763-adf70d4d-cdd7-4aca-88aa-323e10ccfe7e.png)
![Capture d’écran du 2023-02-28 19-01-17](https://user-images.githubusercontent.com/1730702/221940758-79c30daa-b41a-4dda-bd19-72fd30198605.png)

**Comment by wiinxt** [comment]
Elle est en 10 ème position, faudrait voir pourquoi

**Comment by johangirod** [comment]
Par ailleurs, lorsque l'on recherche « bar » l'activité débit de boissons apparaît en 8ᵉ position quand bien même elle contient le mot clé « Bar » isolé dans un tag séparé d'activité principale.

edit : bizarrement, ce comportement n'arrive pas sur Chrome, mais seulement sur FF :thinking:

**Comment by wiinxt** [comment]
J'ai corrigé le bug de Johan.

Par contre pour le problème de Maxime c'est du au 's' de `informatiques` du coup tout les sans 's' monte au dessus
Il faudrait peut être mettre une priorité plus élevé sur le titre ? Actuellement il n'y a pas de différence de score entre titre et texte

J'ai testé mais par exemple pour `velo` ca monte un peut les résultats avec `développement` dans le titre.

**Comment by johangirod** [comment]
Ce problème du `s` est également présent dans le cas suivant :  `conseil en placement financier` marche mais `conseil en placements financiers` ne retourne rien. Y a-t-il un moyen d'ajouter une légère « fuziness » au moteur de recherche (question très naïve dans sa formulation) ?
